Fix #8487: Fixing wrong placement of checkbox

Vinit-Dantkale opened the pull request:
## Explanation
This PR attempts to fix #8487 by adding some CSS to directive.
![image](https://user-images.githubusercontent.com/32016324/73125958-ea037a80-3fd2-11ea-8a88-8159b67b82a6.png)

## Checklist
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The linter/Karma presubmit checks have passed.
  - These should run automatically, but if not, you can manually trigger them locally using `python -m scripts.pre_commit_linter` and `python -m scripts.run_frontend_tests`.
- [x] The PR is made from a branch that's **not** called "develop".
- [] The PR has an appropriate "CHANGELOG: ..." label (If you are unsure of which label to add, ask the reviewers for guidance).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
- [x] The PR addresses the points mentioned in the codeowner checks for the files/folders changed. (See the [codeowner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).)
- [ ] The PR is **assigned** to an appropriate reviewer.
  - If you're a new contributor, please ask on [Gitter](https://gitter.im/oppia/oppia-chat) for someone to assign a reviewer and don't tick this checkbox.
  - If you're not sure who the appropriate reviewer is, please assign to the issue's "owner" -- see the "talk-to" label on the issue. Do not only request the review but also add the reviewer as an assignee.

Commit:
Fix checkbox alignment

Removing unwanted space

Co-authored-by: Parth Patil <parthvin@gmail.com>(Parth1811)

nishantwrp reviewed (changes requested):
![Screenshot_20200126_050002](https://user-images.githubusercontent.com/36989112/73128682-061a1280-3ff9-11ea-84a2-c381e4691287.png)
Hi @Vinit-Dantkale, Thanks for the PR. This is how the question editor in skills dashboard looks now. I think `Solicit Answers` in the middle looks weird.

nishantwrp commented:
@Vinit-Dantkale can you please add a screen shot of how the checkbox look like now. Thanks!

Vinit-Dantkale commented:
Sure
![image](https://user-images.githubusercontent.com/32016324/73186549-ee03d980-4145-11ea-8c53-26437f41c974.png)

nishantwrp commented:
@Vinit-Dantkale did you see how the checkbox looks like in various screen sizes and other pages as well?

Vinit-Dantkale commented:
@nishantwrp yeah, I did check it. Can you also please check it? Please check for commit `f33bb1e` and `a9526c7`.

nishantwrp commented:
Hi @Vinit-Dantkale,
Thanks for the changes. Sorry for the delay.
The checkbox looks perfectly fine and better than the current checkbox in most of the screen sizes.
But in one case the checkbox again appears to be in the center. Please PTAL!
![Screenshot_20200212_035613](https://user-images.githubusercontent.com/36989112/74285442-ddf61780-4d4b-11ea-8272-ca5317fba5a3.png)

Vinit-Dantkale commented:
No problem, I'll look into this. Just caught up with midsem.

On Wed, Feb 12, 2020 at 3:58 AM Nishant Mittal <notifications@github.com>
wrote:

> Assigned #8491 <https://github.com/oppia/oppia/pull/8491> to
> @Vinit-Dantkale <https://github.com/Vinit-Dantkale>.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/oppia/oppia/pull/8491?email_source=notifications&email_token=AHUIPRAIBQ7FYFBGJ2ZYYNTRCMRBBA5CNFSM4KLSMX22YY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZLOORPWSZGOWSBQLXA#event-3028485596>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHUIPRBCDDHBUQ76YL7O3SLRCMRBBANCNFSM4KLSMX2Q>
> .
>

-- 
 <https://www.somaiya.edu>  .                           
<http://www.somaiya-ayurvihar.org>  <http://www.nareshwadi.org>  
<http://www.somaiya.org>  <http://www.helpachild.org.in>  
<http://www.nareshwadi.org>

nishantwrp commented:
Hi @Vinit-Dantkale,
any updates on this one?

Vinit-Dantkale commented:
I fixed for `Kindle Fire HDX` . On what other screens can it cause problem (by intuition), so that I can fix in a go. 
![image](https://user-images.githubusercontent.com/32016324/74969353-71150880-5442-11ea-9734-b63cdc3819a3.png)

Vinit-Dantkale commented:
Sorry for being late bro

nishantwrp reviewed (approved):
lgtm ui wise. @aks681 please do the code review.

aks681 commented on core/templates/dev/head/components/state-editor/state-responses-editor/state-responses.directive.html:
```diff
@@ -1,10 +1,20 @@
+          </strong>
+        </td>
+        <td class="col-md-5">
+          <span ng-if="!isCurrentInteractionTrivial()" ng-hide="!enableSolic  itAnswerDetailsFeature" class="solicit-answer-details-span">
```
Remove the space in 'enableSolicitAnswerDetailsFeature'.

aks681 reviewed (commented):
Just one typo

kevinlee12 commented:
@Vinit-Dantkale can you please restart the CircleCI tests? thanks!

aks681 reviewed (approved):
lgtm!

aks681 reviewed (changes requested):
PR lgtm, but blocking merge till #8693 is merged.

Vinit-Dantkale commented:
Oh, Im so sorry for this. I tried rebase my remote repo and solved conflicts. So sorry, didn't think this would happen.

Vinit-Dantkale commented:
hey @nishantwrp , please have a look I have fixed the merge errors and rebased the branch with the current develop. @parth1811 has helped me with the rebasing.

nishantwrp commented:
@Vinit-Dantkale Ok, @aks681 Please have a look. I think we can merge this one now.

Parth1811 commented:
> hey @nishantwrp , please have a look I have fixed the merge errors and rebased the branch with the current develop. @Parth1811 has helped me with the rebasing.

@Vinit-Dantkale thanks for the credit, I was happy to help :)

kevinlee12 commented:
@Vinit-Dantkale please restart your circleci tests.

DubeySandeep commented:
@Vinit-Dantkale Please merge your PR with upstream/develop to fix the flaky e2e test issue on CircleCI (if it's a flaky error).
